openstack/baremetalintrospection/v1/introspection/testing/results_test.go

```diff
@@ -88,6 +88,22 @@ func TestExtraHardware(t *testing.T) {
 	}
 }
 
+func TestNumaTopology(t *testing.T) {
+	numatopology := `{
```

Comment:
Instead of having the JSON fixture embedded in the test function, it should go in the [`fixtures.go`](https://github.com/gophercloud/gophercloud/blob/master/openstack/baremetalintrospection/v1/introspection/testing/fixtures.go) file.

Also, can you confirm that this fixture was pulled from a running installation/environment either from a production environment or devstack? You should be able to pull this data by using the `--debug` flag in most openstack tools, but I'm not familiar with the introspection service to know if it supports `--debug`.